content/articles/how-edge-artificial-intelligence-is-revolutionizing-mobile-application-development/index.md

```diff
@@ -0,0 +1,85 @@
+### How Edge Artificial Intelligence is Revolutionizing Mobile Application Development
+
+The explosive growth of artificial intelligence has influenced app development in recent years. AI and machine learning have matured to provide flexible algorithms for intuitive and seamless experiences. Developers, users, and businesses now have a whole new view of intelligent interactions within mobile apps.
+
+They have embraced the current trend in the mobile app industry dominated by the Internet of Things (IoT) and mobile computing applications. IoT and mobile devices generate loads and loads of data at the network edge. Collecting such massive amounts of data in cloud data centers comes with a fair share of challenges, such as high network bandwidth and latency usage.
+
+The solution to fully unleash the potential of big data and avoid issues associated with cloud computing is to bring AI to the network edge. This is what edge artificial intelligence does by combining AI and edge computing.
+
+This article looks at artificial intelligence at the edge, discusses how it is changing the mobile application development landscape and gives examples of intelligence edge.
+
+### Artificial intelligence at the edge
```

Comment:
> In this place, please add the headings:
> **Artificial Intelligence**
> **Edge Computing** each of this should contain at least two paragraphs.
> 
> This is because some of the people that will be reading this article are not familiar these technical terms.
> 
> **NB** remember to add them to your table of contents.

There are several articles on eng-ed talking about AI and edge computing. Writing about this would be a duplication of content.

Reply:
@BashiirIsla Please embed link to such articles withing your article for the reader.